addons/payment_stripe_sca/models/payment.py

```diff
@@ -85,8 +85,8 @@ def _create_setup_intent(self, kwargs):
         return res
 
     @api.model
-    def _get_stripe_api_url(self):
-        return "https://api.stripe.com/v1/"
+    def _get_stripe_sca_api_url(self):
+        return "https://%s/" % self._get_stripe_api_url()
```

Comment:
That's why I wanted to do a url_join:
what about the last `/` you add ? what if get_stripe_api_url return a url that already ends with it ?

Reply:
@kebeclibre Now there is a new version that should handle correctly any `_get_stripe_sca_api_url`.